pkgs/build-support/bintools-wrapper/setup-hook.sh

```diff
@@ -14,7 +15,9 @@ bintoolsWrapper_addLDVars () {
     getHostRoleEnvHook
 
     if [[ -d "$1/lib64" && ! -L "$1/lib64" ]]; then
-        export NIX_LDFLAGS${role_post}+=" -L$1/lib64"
+        varName=NIX_LDFLAGS${role_post}
```

Comment:
I would add `local varName` so it doesn't "leak".  Also it seems like the style is `local var_name`, e.g. akin to `role_post` ?

It took me awhile to figure this code out but I get what it's doing now... it is indeed very dynamic.

I think `export "$var_name"` is sufficient?